test/test_websocket_grid.py

```diff
@@ -0,0 +1,320 @@
+from app.websocket.app import create_app
+from flask import Flask, session, request, json as flask_json
+from flask_socketio import (
+    SocketIO,
+    send,
+    emit,
+    join_room,
+    leave_room,
+    Namespace,
+    disconnect,
+)
+
+import unittest
+import threading
+import multiprocessing
+import grid as gr
+import syft as sy
+import torch as th
+import time
+import requests
+import json
+
+hook = sy.TorchHook(th)
+
+
+gateway_url = "http://localhost:8080"
```

Comment:
Move all this setup to conftest.py (similar to what we have in PySyft).

Reply:
Done! (fe59983)